tekore/_client/api/browse/__init__.py

```diff
@@ -197,7 +197,9 @@ def recommendations(
         """
         Get a list of recommended tracks for seeds.
 
-        Up to 5 seed values may be provided as artists, genres and tracks.
+        .. warning::
+            The total number of seeds provided in ``artist_ids``, ``genres``
+            and ``track_ids`` must be at least 1 and at most 5.
```

Comment:
I'm not entirely satisfied with the wording here. "At least 1 and at most 5" is a bit awkward, although it surely is correct. "Between 1 and 5" would be better but more ambiguous. Thoughts @oroddlokken?

Reply:
I get what you mean. I think I prefer "must be at least".

Reply:
Yeah alright, let's go with this then!